src/viz-parole-population/VizParolePopulation.js

```diff
@@ -21,6 +46,100 @@ const MapCaption = styled.figcaption`
   font: ${(props) => props.theme.fonts.body};
 `;
 
+const ParoleDemographicsWrapper = styled.figure`
+  height: 64px;
+  width: 376px;
+`;
+
+const ParoleDistrictCountWrapper = styled.div`
+  text-align: right;
+`;
+
+function VizParaoleDistrictCount({ data, currentDistrict }) {
+  if (!currentDistrict) return null;
+
+  const districtData = data[currentDistrict];
+
+  // Each individual district has an "aggregate record" which is defined
+  // by each of its demographic entries being set to "ALL".  Unfortunately,
+  // when no district is selected, there is no "aggregate record" so to get
+  // the population total, we sum the total of the population across a
+  // single dimension, in this case GENDER.
```

Comment:
Ideally, the record that represents the entire state (i.e. the "ALL" district) would also have an "aggregate record", but it does not.  Is this something we can fix upstream so that we can avoid this hack?

Reply:
that would be a question for @jjdressel11

Reply:
Yes! Just updated the query, and the files in GCS now have the district = ALL rows!

Let me know if there are any other places you are having to aggregate on the front end, because I can almost certainly send you those number pre-calculated.

Reply:
Awesome!  Thanks @jjdressel11.  Just to be clear, that means there's now a record in the file that looks something like this, right?

```json
  {
    "state_code": "US_ND",
    "total_supervision_count": "934",
    "supervision_type": "PAROLE",
    "district": "ALL",
    "race_or_ethnicity": "ALL",
    "gender": "ALL",
    "age_bucket": "ALL"
  },
```

Notice that all of the dimension keys (i.e. `race_or_ethnicity`, `gender`, and `age_bucket`) are all set to `ALL` for the `ALL district` record.

Reply:
Yes, here's the exact record (with staging data):

```
{"state_code":"US_ND","supervision_type":"PAROLE","district":"ALL","race_or_ethnicity":"ALL","gender":"ALL","age_bucket":"ALL","total_supervision_count":"934"}
```

Reply:
Boom!  That's perfect.  Thanks, @jjdressel11!